src/bootstrap/Cargo.toml

```diff
@@ -34,8 +34,8 @@ build_helper = { path = "../tools/build_helper" }
 cmake = "0.1.38"
 fd-lock = "3.0.8"
 filetime = "0.2"
-getopts = "0.2.19"
 cc = "1.0.69"
+clap = { version = "4.1.4", features = ["std", "usage", "help", "derive", "error-context"], default-features = false}
```

Comment:
How bad would it be to manually implement rather than use the derive macro to reduce the compile time impact?

Reply:
I don't think we should focus so heavily on the compile times. If they're an issue, we should finally fix #99989 , which is very close to being done.